Assets/Scripts/UI/ItemButton.cs

```diff
@@ -14,6 +15,20 @@ public class ItemButton : MonoBehaviour
 
         public Image Icon => icon;
         public TextMeshProUGUI NameLabel => nameLabel;
+        
+        [HideInInspector]
+        public Item _item;
+
+        [HideInInspector]
+        public InventoryController _inventory;
+
+        private Button _button;
+
+        private void Awake()
+        {
+            _button = GetComponent<Button>();
```

Comment:
I think atm there is no button on the ItemButton Prefab. It could definitely cause some problems with the dragging and sorting of items. If that's the case, let me know:)

Reply:
I can have a look into this later today